mgplvm/rdist/relie.py

```diff
@@ -133,6 +143,8 @@ def __init__(self,
         intialization [optional] : str
             string to specify type of initialization
             ('random'/'PCA'/'identity' depending on manifold)
+        mu [optional] : np.ndarray
```

Comment:
@KrisJensen  I think this will look better in the docs if we do
```python
mu : Optional[nd.ndarray]
```

Reply:
happy with that